app/models/place_bid.rb

```diff
@@ -35,65 +32,26 @@ def unsaveable_bid
 
   private
 
-  def auction
-    @auction ||= Auction.find(params[:auction_id])
-  end
-
   def auction_available?
-    Presenter::Auction.new(auction).available?
+    auction.available?
```

Comment:
Yes, I want the PlaceBid to just get an object. It was a `Policy::Auction`, but I think it makes more sense to define an `AuctionSession` that wraps the policy for a specific Auction type and the current user and can do the bid validation